PoC-differ/differ.go

```diff
@@ -179,54 +207,49 @@ func processClusters(ruleContent map[string]types.RuleContent, impacts types.Glo
 		var deserialized types.Report
 		err = json.Unmarshal([]byte(report), &deserialized)
 		if err != nil {
-			log.Err(err).Msg("Deserialization error")
+			log.Err(err).Msg("Deserialization error - Couldn't create report object")
 			os.Exit(ExitStatusStorageError)
 		}
 
-		log.Info().Msg(separator)
-		log.Info().Msg("Preparing Kafka producer")
-
-		notifier := setupNotificationProducer(kafkaConfig)
-		log.Info().Msg("Kafka producer ready")
-
-		waitForEnter()
+		if len(deserialized.Reports) != 0 {
```

Comment:
probably the opposite condition would be better, as "happy path" in Go points down, not to the left

Reply:
Done! Thanks for the tip